src/common/models/webhook.go

```diff
@@ -0,0 +1,121 @@
+package models
+
+import (
+	"encoding/json"
+	"time"
+)
+
+const (
+	// WebhookPolicyTable is table name for webhook policies
+	WebhookPolicyTable = "webhook_policy"
+	// WebhookJobTable is table name for webhook job
+	WebhookJobTable = "webhook_job"
+)
+
+// WebhookPolicy is the model for a webhook policy.
+type WebhookPolicy struct {
+	ID           int64        `orm:"pk;auto;column(id)" json:"id"`
+	Name         string       `orm:"column(name)" json:"name"`
+	Description  string       `orm:"column(description)" json:"description"`
+	ProjectID    int64        `orm:"column(project_id)" json:"project_id"`
+	TargetsDB    string       `orm:"column(targets)"`
```

Comment:
what are targetDB and hooktypeDB?

Reply:
> what are targetDB and hooktypeDB?

Unify  webhook model between dao layer and data processing layer。came up in last sync Meeting.  
those two props with type string is the data stored into database